core/api/rpc/ws.hpp

```diff
@@ -6,14 +6,24 @@
 #ifndef CPP_FILECOIN_CORE_API_RPC_WS_HPP
 #define CPP_FILECOIN_CORE_API_RPC_WS_HPP
 
+#include <boost/beast/http.hpp>
 #include "api/api.hpp"
 
 namespace boost::asio {
   class io_context;
 }  // namespace boost::asio
 
 namespace fc::api {
+  namespace beast = boost::beast;
+  namespace http = beast::http;
```

Comment:
beast namespace is never used in this file. I suggest to define only:
namespace http = boost::beast::http

Reply:
Fixed